docs/dev-guide/react-native-sdk.md

```diff
@@ -9,7 +9,7 @@ in a customizable way which you can embed in your React Native apps.
 ## Sample application using the React Native SDK
 
 If you want to see how easy integrating the Jitsi React Native SDK into a React Native application is, take a look at the<br/>
-[sample applications repository](https://github.com/jitsi/jitsi-meet-sdk-samples/react-native).
+[sample applications repository](https://github.com/jitsi/jitsi-meet-sdk-samples).
```

Comment:
@mihhu 
Certainly, I can update the link directly to the corresponding folder: [react-native](https://github.com/jitsi/jitsi-meet-sdk-samples/tree/master/react-native).

However, I also recommend considering a consistent approach for the links on the [ios-sdk](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-ios-sdk) and [android-sdk](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-android-sdk) pages. This will ensure a cohesive user experience while navigating through the various SDK samples.

Feel free to let me know your thoughts on this

Reply:
I see what you mean! Since IOS has multiple entries and all, I think linking the id in the page would be the better solution (e.g.. https://github.com/jitsi/jitsi-meet-sdk-samples#react-native). Could you please apply the change for all of them?

Reply:
See your point. Got you and fixed using you solution. Please review

Reply:
Hmm, that's odd, I can't see them. Can you please check that you've pushed them?

Reply:
@mihhu You're right. I forgot to push - my bad. 
Now its pushed and ready to be reviewed